modules/vpc/variables.tf

```diff
@@ -21,6 +21,12 @@ variable "health_check_ip_cidr_ranges" {
   type        = list(string)
 }
 
+variable "install_dashboard_tcp_port" {
+  default     = "8085"
```

Comment:
I'm assuming this default is coming from limitations on GCP routing? since we can't use the 8800 port.

Reply:
That is correct! I've expanded the documentation in https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/62/commits/e2662778f923ea49961da05e3cac2bd335d7bf19

Reply:
looks great!